### PR TITLE
Add quick links to logs in tasks table subview

### PR DESCRIPTION
**Pull request by tejasmanohar**
@tpetr here's a new PR w/ cleaner commits

### CODE REVIEW

File: SingularityUI/app/templates/requestDetail/requestActiveTasks.hbs [hunk 1]
```diff
@@ -36,6 +36,9 @@
                     <td class="hidden-xs" data-value="{{ taskId.updatedAt }}">
                         {{timestampFromNow updatedAt}}
                     </td>
+                    <a href="{{ appRoot }}/task/{{ taskId.id }}/tail/{{ subtituteTaskId ../../config.runningTaskLogPath taskId.id }}" title="Log">
```
Comment (tpetr) [inline SingularityUI/app/templates/requestDetail/requestActiveTasks.hbs]: typo here in the helper name

Comment (tejasmanohar) [inline SingularityUI/app/templates/requestDetail/requestActiveTasks.hbs]: oops, forgot to fix that when copying from the diffs- f219a603eb35e70fb8e9e0e6bb5a9efd990eca84

**Comment by tejasmanohar** [comment]
@tpetr I seem to have resolved the issue in 37237ce7c224c24a9b00a90590a24d6707c29b72. It also seems to be the root cause of my frequent server crashes. Would you please merge this commit into the `hs_staging` branch?

**Comment by ssalinas** [comment]
@tejasmanohar put the updates in hs_staging for you

**Comment by tejasmanohar** [comment]
@ssalinas thanks

**Comment by tejasmanohar** [comment]
@tpetr would you please merge this latest commit c947f2e073d4b258738b499dadaf125f1a5341ad into hs_staging? it wraps the <a> in a <td>

**Comment by tejasmanohar** [comment]
@tpetr It looks like this wasn't merged properly. Is it possible to "revert" the merge through the GitHub interface or must it be done manually? https://help.github.com/articles/reverting-a-pull-request/

I think the issue was caused by merging https://github.com/HubSpot/Singularity/pull/560 partially manually before (guessing by the `hs_staging` label there)- maybe you can revert that as well?

**Pull request merged**